accepted/targeting-packs-and-runtime-packs.md

```diff
@@ -0,0 +1,286 @@
+# Background
+
+The objectives here are:
+
+1. Summarize the new approach of FrameworkReferences, targeting packs and runtime packs
+2. Establish details such as package naming, layout and versioning
+
+Implementation progress is tracked by https://github.com/dotnet/cli/issues/10048, with [ZenHub](https://www.zenhub.com) required to see the child issue links.
+
+
+# Historical approaches
+
+To date, there have effectively been two different approaches to framework references:
+
+**1. Traditional targeting packs**
+* Used by "classic" frameworks (.NET Framework, Silverlight, Windows Phone, Windows 8)
+* Contain only reference assemblies, xml documentation, and framework list
+* Installed to C:\Program Files (x86)\Reference Assemblies\[Target Framework]\[Version]
```

Comment:
The path actually has an additional `Microsoft` folder: `C:\Program Files (x86)\Reference Assemblies\Microsoft`